index.html

```diff
@@ -17,41 +17,30 @@
     })(window,document,'script','dataLayer','GTM-5N6WR7G');</script>
 
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/make-sense-ico.png" />
+
+    <link rel="shortcut icon" href="/make-sense-ico.png" />
     <link href="https://fonts.googleapis.com/css?family=Saira+Semi+Condensed&display=swap" rel="stylesheet">
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    
+    <meta name="theme-color" content="#171717" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <link rel="manifest" href="/manifest.json" />
     <title>Make Sense</title>
   </head>
   <body>
+    <style>
+      body {
+        background-color: #171717;
+      }
+    </style>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <noscript>
       <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5N6WR7G" height="0" width="0" style="display:none;visibility:hidden"></iframe>
     </noscript>
     <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
+    <script type="module" src="/src/index.tsx"></script>
```

Comment:
Could you let me know why do we need to add that script tag here? Does that have anything to do with migrating to vite?

Reply:
yes, vite uses Esm Modules by directly load scripts. you can [learn more here](https://darekkay.com/blog/create-react-app-to-vite/)

Reply:
Thanks for the link! I understand more about that migration process now.